<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Replace</class>
 <widget class="QDialog" name="Dialog_Replace">
  <property name="modal">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>223</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <item row="3" column="1">
    <widget class="QCheckBox" name="matchCase_CKB">
     <property name="text">
      <string>Match Case</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2" colspan="4">
    <layout class="QGridLayout" name="gridLayout_1">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Find:</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Replace With:</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="find_Combo">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="replace_Combo">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="16" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_20">
     <item>
      <spacer name="horizontalSpacer_20">
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="replace_PB">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_21">
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="replaceAll_PB">
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_22">
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>25</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_PB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_23">
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="9" column="2">
    <spacer name="verticalSpacer_2">
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="wholeWords_CKB">
     <property name="text">
      <string>Match Whole Words</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_1">
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>8</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>wholeWords_CKB</tabstop>
  <tabstop>replaceAll_PB</tabstop>
  <tabstop>cancel_PB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
